charts/rucio-server/values.yaml

```diff
@@ -148,6 +148,29 @@ ftsRenewal:
       cpu: 100m
       memory: 128Mi
 
+policyPackages:
+  enabled: false
+  # Make sure the trailing slash is present
+  mountPath: /opt/policy_packages/
+  # Use underscores instead of hyphens for module names
+  # (e.g. atlas_rucio_policy_package instead of atlas-rucio-policy-package)
+  packages: []
+#    # example: install from an index (default is PyPI)
+#    - moduleName: vo_1_policy_package
+#      requirement: vo_1_policy_package==1.4.0
+#      version: 1.4.0
+#    # example: install from a git repository
+#    - moduleName: vo_2_policy_package
+#      requirement: git+https://github.com/vo-2/vo-2-policy-package@v0.1.0
+#      version: 0.1.0
```

Comment:
Comments not following the expected indentation level looks out of place to me.

Reply:
Addressed in latest revision